config/bps_idf.yaml

```diff
@@ -1,46 +1,25 @@
-includeConfigs:
-- pipelines_check_idf.yaml
-
-
 #PANDA plugin specific settings:
 idds_server: "https://aipanda015.cern.ch:443/idds"
 placeholderParams: ['qgraphNodeId', 'qgraphId']
+defaultPreCmdOpts: "--long-log --log-level=VERBOSE --log-file payload-log.json"
```

Comment:
This is the line I commented in slack about.  defaultPreCmdOpts is in every pipetask and butler command line.  This means those commands that are executed during submit will also output that json file.  Currently it looks like at least one command is executed while in the cwd and then at least one after switching to the run directory.  This is something that probably could be changed in bps, but I'd rather not rush to get it in tonight.

Reply:
I find the side effect of having payload-log.json in cwd inconsistent with the rest of artifacts.  But can live with it for now.